Meme/express.js

```diff
@@ -0,0 +1,31 @@
+const fs = require('fs');
+const express = require('express');
+const bp = require('body-parser');
+const app = express();
+const gm = require('gm');
+
+app.listen(3608);
+app.use(express.static('public'));
+app.use(bp.json({limit: "500mb"}));
+
+const appendTextToFile = (req,res) =>{
+  const chatMessage = `${req.body.name}: ${req.body.comment}\n`;
+  const textFilePath = '/home/alopez/garagescript/Meme/public/book.txt';
+
+  fs.appendFile(textFilePath, chatMessage,()=>{});
+  res.send('append comment to /book.txt');
+}
+app.post('/send', appendTextToFile);
+
+const createImageMeme = (req, res) =>{
+  const imagePath = `/home/alopez/garagescript/Meme/public/${req.body.name}.png`;
+  const imageData = req.body.Canvas.replace('data:image/png;base64',""); 
```

Comment:
I prefer this to be done in the client side (browser). Its more efficient this way.

Reply:
I'm not sure how to do that. This is the way you taught us.